apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.tsx

```diff
@@ -464,11 +465,11 @@ const MainSideBar = () => {
                     collapsed={secondAnim}
                     NotifComponent={
                       // @ts-expect-error TODO doesn't exist yet in our types
-                      ptxEarnConfig?.params?.isNew ? (
+                      ptxEarnConfig?.params?.isNew && (
                         <CustomTag active type="plain" size="small">
                           {t("common.new")}
                         </CustomTag>
-                      ) : null
+                      )
```

Comment:
As you repeat this multiple times and we maybe want to have is on others entries in a near future 
You can think to create something like `getNotifComponent` that takes `isNew ` in params :) 
 And apply it to each entry when it's needed
 
 it will be more readable i think when you have lot of case